<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FittingParametersWidget</class>
 <widget class="QWidget" name="FittingParametersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fitting Parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,2,0,0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Parameters</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="parameter_tree">
     <property name="editTriggers">
      <set>QAbstractItemView::AllEditTriggers</set>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0">
     <item>
      <widget class="QPushButton" name="remove_group_button">
       <property name="text">
        <string> - </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="group_type_combo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Multi-exponential decay</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FRET decay</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Anisotropy Decay</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_group_button">
       <property name="text">
        <string> + </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Channel Factors</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="channel_factor_tree">
     <property name="editTriggers">
      <set>QAbstractItemView::AllEditTriggers</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
